notebook/static-src/edit/js/main.js

```diff
@@ -95,3 +84,8 @@ require([
     // On document ready, resize codemirror.
     $(document).ready(_handle_resize);
 });
+
+}).catch(function(err) {
+    console.error('Could not load globals', err);
```

Comment:
This seems like one of those times where we error better for the user since they won't know to look at the console. Probably fine for now though.